app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag sessions_path, class: 'sign_in' do
+        %header
+          %h1 Sign in
+        %fieldset
+          .form-group
+            = label_tag :email, "Email Adress"
+            = email_field_tag :email
+            = label_tag :password
+            = password_field_tag :password
+        %fieldset.form-group.action
+          %input(type="submit" value="Sign in" class="btn btn-default")
```

Comment:
This should rather be `submit_tag ...` like the rest of the form to be consistent.